presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -49,26 +48,32 @@
         this.executor = requireNonNull(executor);
     }
 
-    public File storeConfigFiles(Path configDir)
+    /**
+     * Save the given directory in a tarball
+     */
+    public Path storeConfigFiles(Path configDir)
```

Comment:
I've renamed it to `backupDirectory`.

This way, if we decide to change it to not use `tar`, it will still make sense.

I've also added documentation.